gtk/DetailsDialog.cc

```diff
@@ -1936,22 +1983,26 @@ void appendScrapeInfo(tr_tracker_view const& tracker, time_t const now, Gtk::Tex
 
         if (tracker.lastScrapeSucceeded)
         {
-            gstr << gtr_sprintf(
-                _("Tracker had %s%'d seeders and %'d leechers%s %s ago"),
-                success_markup_begin,
-                tracker.seederCount,
-                tracker.leecherCount,
-                success_markup_end,
-                timebuf);
+            gstr << fmt::format(
+                // {markup_begin} and {markup_end} should surround the seeder/leecher text
+                _("Tracker had {markup_begin}{seeder_count} {seeders_ngettexted} and {leecher_count} {leechers_ngettexted}{markup_end} {time_span} ago"),
+                fmt::arg("seeder_count", tracker.seederCount),
+                fmt::arg("seeders_ngettexted", ngettext("seeder", "seeders", tracker.seederCount)),
+                fmt::arg("leecher_count", tracker.leecherCount),
+                fmt::arg("leechers_ngettexted", ngettext("leecher", "leechers", tracker.leecherCount)),
+                fmt::arg("time_span", timebuf),
+                fmt::arg("markup_begin", SuccessMarkupBegin),
+                fmt::arg("markup_end", SuccessMarkupEnd));
         }
         else
         {
-            gstr << gtr_sprintf(
-                _("Got a scrape error \"%s%s%s\" %s ago"),
-                err_markup_begin,
-                tracker.lastScrapeResult,
-                err_markup_end,
-                timebuf);
+            gstr << fmt::format(
+                // {markup_begin} and {markup_end} should surround the error text
+                _("Got a scrape error {markup_begin}{error}{markup_end} {time_span} ago"),
```

Comment:
Maybe still quote the error since it's in the middle of the phrase? Alternatively, mark it up with more than just color (e.g. `<em>`).

Reply:
4665ebc22aec09a40dbbcdf1e9d276c33db46d0f